test/test_dc_powerflow.jl

```diff
@@ -28,8 +28,7 @@
     # CASE 1: ABA and BA matrices
     solved_data_ABA = solve_powerflow(DCPowerFlow(), sys)
     @test isapprox(
-        solved_data_ABA["1"]["flow_results"].P_from_to +
-        solved_data_ABA["1"]["flow_results"].P_to_from,
+        solved_data_ABA["1"]["flow_results"].P_from_to,
```

Comment:
Shouldn't we also test that the value of `P_to_from` is the opposite of `P_from_to`?